lib/modules/manager/asdf/extract.ts

```diff
@@ -0,0 +1,40 @@
+import { logger } from '../../../logger';
+import { isSkipComment } from '../../../util/ignore';
+import { GithubTagsDatasource } from '../../datasource/github-tags';
+import type { PackageDependency, PackageFile } from '../types';
+
+const upgradeableTooling: Record<
+  string,
+  Pick<PackageDependency, 'depName' | 'datasource' | 'packageName'>
+> = {
+  nodejs: {
+    depName: 'node',
+    datasource: GithubTagsDatasource.id,
+    packageName: 'nodejs/node',
+  },
+};
+
+export function extractPackageFile(content: string): PackageFile {
+  logger.trace('asdf.extractPackageFile()');
+
+  const regex =
+    /^(?<content>(?<toolname>(\w+))\s+(?<version>(\d[\d.]+\d)))([^#]*(#(?<comment>(.*))))?/gm;
```

Comment:
```suggestion
    regEx(/^(?<content>(?<toolname>(\w+))\s+(?<version>(\d[\d.]+\d)))([^#]*(#(?<comment>(.*))))?/gm);
```

use our regex wrapper